apparmor.d/profiles-s-z/tlp

```diff
@@ -0,0 +1,117 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2021-2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Barmogund <set508@proton.me>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/tlp
+profile tlp @{exec_path} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/bus-system>
+  include <abstractions/disks-read>
+  include <abstractions/graphics>
+  include <abstractions/bus/org.freedesktop.PolicyKit1>
+  include <abstractions/nameservice-strict>
+  include <abstractions/perl>
+
+  capability dac_read_search,
+  capability net_admin,
+  capability sys_nice,
+  capability sys_rawio,
+  capability sys_tty_config,
+
+  network netlink raw,
+
+  ptrace read peer=unconfined,
+
+  @{exec_path} mr,
+
+  @{bin}/systemctl              rix,
```

Comment:
In a subprofile: https://apparmor.pujol.io/development/abstractions/#appsystemctl